python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_worker.py

```diff
@@ -126,32 +126,38 @@ def setup_distributed(self, mode, cluster_info, driver_ip, driver_tcp_store_port
     def train_epochs(self, data_creator, epochs=1, batch_size=32, profile=False,
                      wrap_dataloader=None, callbacks=None,
                      validation_data_creator=None):
-        self.load_state_dict(self.state_dict.value)
+        if self.state_dict:
+            self.load_state_dict(self.state_dict.value)
```

Comment:
Will this get improvement?
Since for the logic in estimator, if there's model creator, there will always be self.state_dict?

Reply:
`self.state_dict` will be none if `epoch == 0`.